WebDriverAgentLib/Categories/XCUIElement+FBWebDriverAttributes.m

```diff
@@ -191,6 +191,15 @@ - (BOOL)isWDVisible
   return [[self fb_cachedValueWithAttributeName:@"isWDVisible" valueGetter:getter] boolValue];
 }
 
+#if TARGET_OS_TV
```

Comment:
Will remove since hasFocus is included in iPhone as well.
(I noticed after this change...)
https://developer.apple.com/documentation/xctest/xcuielementattributes/1627636-hasfocus